test/test-conditions-2.js

```diff
@@ -0,0 +1,6 @@
+const operations = require("./operations-conditions-2.js");
+const assert = require("assert");
+
+it("Выдает правильный язык conditions-task-2", () => {
```

Comment:
you have tested only one possible input, you should test all of them in this case

Reply:
it's a duplicated test

Reply:
What do you mean? Am I did test wrong or what?